test/extensions/filters/http/kill_request/crash_integration_test.cc

```diff
@@ -68,9 +73,10 @@ TEST_P(CrashIntegrationTestAllProtocols, UnwindsTrackedObjectStack) {
   //  - ActiveStream
   //  - Http(1|2)::ConnectionImpl
   //  - Network::ConnectionImpl
-  const std::string death_string = GetParam().downstream_protocol == Http::CodecType::HTTP2
-                                       ? "ActiveStream.*Http2::ConnectionImpl.*ConnectionImpl"
-                                       : "ActiveStream.*Http1::ConnectionImpl.*ConnectionImpl";
+  const std::string death_string =
+      GetParam().downstream_protocol == Http::CodecType::HTTP2
+          ? "[\\s\\S]*ActiveStream[\\s\\S]*Http2::ConnectionImpl[\\s\\S]*ConnectionImpl[\\s\\S]*"
```

Comment:
This is messy. Just add `(?s)` to the start of the regex to restore the previous behavior where `.` matchs newline.

https://github.com/envoyproxy/envoy/pull/36699/commits/5cc0564da3b39542689b193c9ffe337eef5251c8
(part of #36699)

Reply:
Done